sbncode/CAFMaker/FillReco.cxx

```diff
@@ -111,6 +111,46 @@ namespace caf
     srtrack.hitb.plane = track.plane2;
   }
 
+  void FillOpFlash(const recob::OpFlash &flash,
+                  int InputCryostat, 
+                  caf::SROpFlash &srflash,
+                  bool allowEmpty) {
```

Comment:
This parameter doesn't do anything, remove it.

Reply:
More of me following status quo. So many functions in the file had this `allowEmpty` that I thought there must be some subtle need for it that I was missing somewhere. Happy to cut it if it's actually useless.